libraries/core-react/src/components/Card/Card.tokens.ts

```diff
@@ -17,7 +18,21 @@ const {
   },
 } = tokens
 
-export const card = {
+export type CardToken = ComponentToken & {
+  shape: {
+    minHeight: string
+    minWidth: string
+    borderRadius: string
+  }
```

Comment:
Should we have `shape` instead of `height`/`width` on `ComponentToken`? We are making our own deviation here with something that maybe we need to expand `ComponentToken` with 🤔

Reply:
Yes, it might be as this was values from the `shape` objects from tokens. But on the other hand, it wasn't used as minHeight and minWidth in code, so I just moved it to height, width and borders accordingly. But let's see if this is the case in other components as well or not. I don't think we use shape that much, we probably have used it more in code 🤔

Reply:
We don't have much min/max, height/width values. Maybe less that we should... We should have support for it in the `ComponentToken` type, question is in what manner 🤔

Reply:
Rome wasn't built in a day 😂

Reply:
![Skjermbilde 2021-03-02 kl  09 55 12](https://user-images.githubusercontent.com/1568252/109623479-77ea5700-7b3d-11eb-8acc-4fdf65189acb.png)
`Card` uses a minHeight, but this doesn't actually makes sense in code as the padding is 32px, so a 4px something added 🤔  But why

Reply:
@mimarz  Instead of adding `minHeight` and `minWidth` to token types because of `Card`, I suggest to remove minHeight and minWidth from Card styles. 😨  A min width of 112px (magic px number 😓 ) doesn't actually work anyway, see image of a Card with width of 112px
![Skjermbilde 2021-03-02 kl  10 06 17](https://user-images.githubusercontent.com/1568252/109625146-41154080-7b3f-11eb-8a01-47dacde94482.png)

Reply:
if it doesn't work remove it 😅